.github/issue_template.md

```diff
@@ -0,0 +1,26 @@
+PLEASE FOLLOW THE CHECKLIST BELOW WHEN CREATING A NEW ISSUE. THE CHECKLIST IS
+FOR YOUR INFORMATION AN MUST BE REMOVED BEFORE SUBMITTING ISSUE.
+
+## Checklist
+
+- [ ] Is the issue marked with exactly one of the following labels
+
+    - Bug
+    - Feature
+    - Improvement
+    - infrastructure
+
+- [ ] If the issue reports a bug:
+
+    - [ ] Is it also marked with one of the `Impact - {Minor,Major,Critical}` labels?
+    - [ ] Does the description contain all necessary information to reproduce
+        the error?
+
+- [ ] Is the issue assigned to a milestone? This should be `next` unless you
+    know exactly what you are doing.
```

Comment:
... unless you target a specific release directly
(the other text is a bit condescending)

Reply:
Fixed.